tests/rptest/tests/topic_limits_test.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2025 Redpanda Data, Inc.
+
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.services.cluster import cluster
+from rptest.clients.types import TopicSpec
+from rptest.clients.rpk import RpkTool
+from rptest.clients.kafka_cli_tools import KafkaCliTools
+
+
+class TopicLimitsTest(RedpandaTest):
+    def __init__(self, test_context):
+        super(TopicLimitsTest, self).__init__(
+            test_context=test_context,
+            num_brokers=3,
+            extra_rp_conf={'auto_create_topics_enabled': False})
```

Comment:
this is already the default?

Reply:
Mainly to be explicit about the test changing it and what it's initial value is. Will remove if you feel that it isn't needed though.